doc/tutorials/acrn_on_qemu.rst

```diff
@@ -252,6 +252,7 @@ Bring Up User VM (L2 Guest)
       acrn-dm -A -m $mem_size -s 0:0,hostbridge \
       -s 3,virtio-blk,~/UserVM.img \
       -s 4,virtio-net,tap0 \
+      --cpu_affinity 1 \
```

Comment:
The sample script in the repo also includes the following two lines:
```
-k /home/guestl1/acrn-dm-bins/bzImage_uos \
-B "earlyprintk=serial,ttyS0,115200n8 consoleblank=0 root=/dev/vda1 rw rootwait maxcpus=1 nohpet console=tty0 console=hvc0 console=ttyS0 no_timer_check ignore_loglevel log_buf_len=16M tsc=reliable" \
```

They do not appear in this launch script snippet, does the user need to remove those?

Reply:
Yes, since OVMF will be used to launch User VM.

Reply:
Thanks, please make this clear in the document.